pkg/model/parents.go

```diff
@@ -30,3 +30,6 @@ type Parent struct {
 	ID   iotago.BlockID
 	Type ParentsType
 }
+
+// ParentReferences is a map between parent type and block IDs.
+type ParentReferences map[ParentsType]iotago.BlockIDs
```

Comment:
instead of having this type can't we just have 3 separate lists/maps for each parent type?

Reply:
It will bloat the number of arguments passed around block creation functions and their return values. Isn't this more compact?